kernel/usage.c

```diff
@@ -170,6 +170,13 @@ void z_sched_cpu_usage(uint8_t cpu_id, struct k_thread_runtime_stats *stats)
 }
 #endif /* CONFIG_SCHED_THREAD_USAGE_ALL */
 
+void z_sched_thread_reset_long_frame(struct k_thread *thread)
```

Comment:
Consider `inline` since it is a one line function.

Reply:
Read up on https://www.kernel.org/doc/html/latest/process/coding-style.html#the-inline-disease and you are right.
Sorry for the noise.